### ISSUE TITLE
Аналитика, цели

### ISSUE THREAD

**Issue by iliakan** [iliakan/javascript-nodejs#344]
Какие цели трекаем?

**Comment by Bezart** [comment]
Итак, вот начальны список вещей которые хотелось бы трекать:

**Convertion**
1. Количество успешных покупок PDF учебника
2. Количество попыток покупки учебника / успешных покупок
3. Количество успешных покупок курсов
4. Количество купленных курсо-мест

**Engagement**
1. Количество успешных регистраций
2. Количество успешных регистраций через соц сети
3. Количество успешных регистраций через email
4. Количество успешных регистаций / количество попыток (в случае создания нового аккаунта через почту)
5. Количество посетителей посетивших более 1 страницы за неделю
6. Количество посетителей посетивших более 3 страниц за неделю
7. Количество посетителей посетивших более 5 страниц за неделю
8. Количество посетителей посетивших более 10 страниц за неделю
9. Количество поисков
10. Количество поисков с кликами по резултатам
11. Количество людей совершивших >= 1 поисков за визит
12. Количество людей совершивших >= 3 поисков за визит
13. Количество соц. шеров (нужен общее кол + можно разбить по типам twitter, facebook и т д)
14. Количество полных скачанных скринкастов Node.js (низкое разр)
15. Количество полных скачанных скринкастов Node.js (высокое разр)
16. Количество скачанных индивидуальных частей скринкаста Node.js (низкое разр)
17. Количество скачанных индивидуальных частей скринкаста Node.js (высокое разр)
18. Количество переходов на скринкаст (YouTube плейлист)
19. Количество переходов на скринкаст (YouTube индивидуальный урок)
20. Количество пройденных тестов
21. Количество незавершенных тестов
22. Количество полностью пройденных тестов
23. Количество незаконченных тестов

**Qualitative metrics**
1. Количество пройденных тестов 0—20%
2. Количество пройденных тестов 20—40%
3. Количество пройденных тестов 40—60%
4. Количество пройденных тестов 60—80%
5. Количество пройденных тестов 80—90%
6. Количество пройденных тестов 90—100%

**Technical**
1. HiDPI Desktop пользователи

Еще нужна интеграция с поиском: https://support.google.com/analytics/answer/1012264?hl=en

**Comment by iliakan** [comment]
Трекать - ты имеешь в виду отправлять все эти данные в Google Analytics, чтобы потом строить Funnels?

Будем ли мы их реально использовать? Зачем?

**Comment by Bezart** [comment]
Да, будем их использовать для анализа поведения пользователей и дальнейшей оптимизации сайта. Стандартные репорты аналитики не предоставляют конкретных данных по метрикам, а персональные отзывы очень субъективны. Для того чтобы понимать аудиторию, нужно смотреть на конкретные цели а не на общую статистику.

**Comment by Bezart** [comment]
@iliakan Интеграция поиска по-моему отвалилась, можешь глянуть плиз

**Comment by iliakan** [comment]
как увидеть проблему?
